fabfile.py

```diff
@@ -762,6 +772,13 @@ def reset_security_tokens():
     reset_controller_token()
     config_couchdb()
     print(green('All the tokens have been reset.'))
+    
+    if is_pi():
+        reset_ssh_keys()
```

Comment:
Because we definitely don't want to regenerate a production server set of keys.
For the RaspberryPi we are providing (will provide) an already built image, thus the keys are generated by our side in the first place.